pkg/qemu/qemu.go

```diff
@@ -324,9 +324,9 @@ func Cmdline(cfg Config) (string, []string, error) {
 	default:
 		// QEMU does not seem to support virtio-vga for aarch64
 		args = append(args, "-vga", "none", "-device", "ramfb")
-		args = append(args, "-device", "usb-ehci")
-		args = append(args, "-device", "usb-kbd")
-		args = append(args, "-device", "usb-mouse")
+		args = append(args, "-device", "qemu-xhci,id=usb-bus")
```

Comment:
This requires QEMU >= 2.9.
I guess this is fine, but cc-ing @jandubois FYI
https://github.com/qemu/qemu/commit/72a810f411abaabc55f375533220adf69e059c89

Reply:
> I guess this is fine, but cc-ing @jandubois FYI

Yes, seems fine, and works for me.

But as I mentioned in #499, I cannot repro the failure on macOS on aarch64 with the `default.yaml` config.

And I was also wondering why we need USB devices at all? What happens if we just drop them?

Reply:
> And I was also wondering why we need USB devices at all? What happens if we just drop them?

We will lose keyboard and mouse. Probably negligible for non-GUI mode though.

Reply:
>  Probably negligible for non-GUI mode though.

Do we have a GUI mode? I know we can get a console (with horrible performance impact), but is there a way to actually run a GUI?

On the other hand, I would hope that performance impact from emulating a USB device that is not being used is really minimal as well (really only thinking about battery impact when running on a laptop).

Reply:
By "GUI" I just meant setting `.video.display = "cocoa"`

Reply:
> By "GUI" I just meant setting .video.display = "cocoa"

So maybe we drop the USB devices with the `display: none` setting. I may try that when I'm bored. 😄